client/ayon_substancepainter/plugins/publish/collect_textureset_images.py

```diff
@@ -113,7 +113,8 @@ def create_image_instance(self, instance, template, outputs,
             host_name=context.data["hostName"],
             product_type="texture",
             variant=instance.data["variant"] + suffix,
-            project_settings=context.data["project_settings"]
+            project_settings=context.data["project_settings"],
+            product_base_type=instance.data["productBaseType"]
```

Comment:
I assume this is only present with more recent versions of ayon-core? So should this be `data.get("productBaseType")` @antirotor ?